sdk/eventhub/event-hubs/samples-dev/sendBufferedEvents.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/**
+ * @summary Demonstrates how to send events to an Event Hub from a buffered producer.
+ *
+ * @azsdk-weight 60
+ */
+
+import { EventHubBufferedProducerClient, OnSendEventsErrorContext } from "@azure/event-hubs";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = process.env["EVENTHUB_CONNECTION_STRING"] || "";
+
+async function handleError(ctx: OnSendEventsErrorContext): Promise<void> {
+  console.log(JSON.stringify(ctx.error), undefined, "  ");
```

Comment:
Does the `ctx` hold any info about the event that we failed to send? I think it would be useful to demonstrate that in the sample

Reply:
done!

Reply:
What is the undefined and "  " about?

Reply:
`stringify` 2nd parameter is an optional function that specifies how to stringify certain parts in the input object but since there is nothing special we want to do, `undefined` is passed. The 3rd parameter is used to insert whitespace in the output string for indenting to make the output string more readable.

Reply:
> console.log(JSON.stringify(ctx.error), undefined, "  ")

but... but... the undefined is going as a second parameter to console.log() instead of JSON.stringify() here...

Reply:
ops, I fixed it in https://github.com/Azure/azure-sdk-for-js/pull/18674/commits/863cf930f646be83c1ff065dee6fd9b7ab165ba4